subprojects/docs/src/docs/userguide/introductionDependencyManagement.adoc

```diff
@@ -15,33 +15,45 @@
 
++++++
+<figure xmlns:xi="http://www.w3.org/2001/XInclude">
+    <title>Dependency management big picture</title>
+    <imageobject>
+        <imagedata fileref="img/dependency-management-resolution.png" width="130mm" />
+    </imageobject>
+</figure>
++++++
 
-[[sub:dependency_management_and_migrations]]
-=== Flexible dependency management for migrations
+The example project builds Java source code. Some of the Java source files import classes from link:https://github.com/google/guava[Google Guava], a open-source library providing a wealth of utility functionality. In addition to Guava, the project needs the link:http://junit.org/junit5/[JUnit] libraries for compiling and executing test code.
 
-Dependency management can be particularly challenging during a migration from one build system to another. If you are migrating from a tool like Ant or Maven to Gradle, you may be faced with some difficult situations. For example, one common pattern is an Ant project with version-less jar files stored in the filesystem. Other build systems require a wholesale replacement of this approach before migrating. With Gradle, you can adapt your new build to any existing source of dependencies or dependency metadata. This makes incremental migration to Gradle much easier than the alternative. On most large projects, build migrations and any change to development process is incremental because most organizations can't afford to stop everything and migrate to a build tool's idea of dependency management.
+Guava and JUnit represent the _dependencies_ of this project. A build script developer can <<declaring_dependencies,declare dependencies>> for different scopes e.g. just for compilation of source code or for executing tests. In Gradle, the <<sub:scope_of_dependency_configurations,scope of a dependency>> is called a _configuration_. For a full overview, see the reference material on <<dependency_types,dependency types>>.
 
-Even if your project is using a custom dependency management system or something like an Eclipse .classpath file as master data for dependency management, it is very easy to write a Gradle plugin to use this data in Gradle. For migration purposes this is a common technique with Gradle. (But, once you've migrated, it might be a good idea to move away from a .classpath file and use Gradle's dependency management features directly.)
+Often times dependencies come in the form of one of many files. You’ll need to tell Gradle where to find those files so they can be consumed by the build. The location storing the files of a dependency is called a _repository_. By <<declaring_repositories,declaring repositories>> for a build, Gradle will know how to find and retrieve dependencies. Repositories can come in different forms: as local directory or a remote binary repository. The reference on <<repository_types,repository types>> provides a broad coverage on this topic.
```

Comment:
Do you mean "form of one _or_ many files"?

Reply:
Ooops. That should be an "or" instead of "of". Will fix that.